networks/testnet/markets.json

```diff
@@ -1,6 +1,6 @@
 [
-    "spot://BTC/USDT",
-    "spot://ETH/USDT",
-    "spot://MATIC/USDT",
-    "spot://SHIB/USDT"
+    "spot://USDC/USDT",
+    "spot://LUBE/USDT",
+    "spot://WBTC/USDT",
+    "spot://LINDA/USDT"
```

Comment:
there is not trading activity for this token 
https://coinmarketcap.com/dexscan/ethereum/0x246ba530d046c0ae454b5e962a9c4ede382047bd/